Automattic/sensei
# Translate title, description, keywords of a block

Resolves part of #6351 

This PR is an attempt to "fix symptoms".
It is still not clear for me, why I had to remove title and description from block.json for one blocks, and didn't need to do that for others. Looks like it shadowed them somehow, but I didn't find an answer why.

## Proposed Changes
* Make titles and descriptions for all Sensei blocks translatable.

## Testing Instructions

1. Run `npm run i18n:build && npm run build:assets`
2. Install Loco Translate, change website language.
3. Go to Loco Translate -> Sensei LMS -> Synchronize.
4. Find strings "Quiz" and "Course Actions" and add translations for them.
5. Create a new lesson and open Inserter. Check Quiz and Course Actions use your translation.
6. Go further and translate descriptions for those blocks.
7. Then add these blocks in the lesson (or course) and open the Inspector. Both the block title and the description should use your translations.

## Pre-Merge Checklist
